hmpps_cpr_splink/cpr_splink/visualisation/visualise_cluster_spec.py

```diff
@@ -0,0 +1,229 @@
+from copy import deepcopy
+from typing import Any
+
+NODE_TOOLTIP_FIELDS = [
+    "id",
+    "match_id",
+    "source_system",
+    "source_system_id",
+    "name_1_std",
+    "name_2_std",
+    "name_3_std",
+    "last_name_std",
+    "first_and_last_name_std",
+    "date_of_birth",
+    "cro_single",
+    "pnc_single",
+]
+
+EDGE_COMPARISON_FIELDS = [
+    "id",
+    "match_id",
+    "source_system",
+    "source_system_id",
+    "name_1_std",
+    "name_2_std",
+    "name_3_std",
+    "last_name_std",
+    "first_and_last_name_std",
+    "date_of_birth",
+    "cro_single",
+    "pnc_single",
+]
+
+NODE_TOOLTIP_SIGNAL = "{" + ", ".join(f'"{field}": datum.{field}' for field in NODE_TOOLTIP_FIELDS) + "}"
+
+EDGE_TOOLTIP_SIGNAL = (
+    "{" + ", ".join(f'"{field}": datum.{field}_l + " <-> " + datum.{field}_r' for field in EDGE_COMPARISON_FIELDS) + "}"
+)
+
+# see https://www.robinlinacre.com/microblog/#different-ways-of-setting-out-data-in-a-vega-force-directed-layout
+BASE_SPEC: dict[str, Any] = {
```

Comment:
I hard coded this rather than put it in a `.json` document so we didn't have to mess about with like `importlib.resources`